app/(service)/recruit-detail/[shopId]/[recruitId]/_components/recruitCard.tsx

```diff
@@ -0,0 +1,104 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+
+import Image from 'next/image';
+import { useParams } from 'next/navigation';
+
+import Button from '@/components/button/button';
+import PercentageBadge from '@/components/post/percentageBadge';
+import apiClient from '@/libs/axios';
+import calculatePercentage from '@/utils/calculatePercentage';
+import formatDateRange from '@/utils/formatDateRange';
+
+interface FetchData {
+  hourlyPay: 'number';
+  startsAt: 'string';
+  workhour: 'number';
+  description: 'string';
+  closed: 'boolean';
+  shop: {
+    item: {
+      name: 'string';
+      category: 'string';
+      address1: 'string';
+      description: 'string';
+      imageUrl: 'string';
+      originalHourlyPay: 'number';
+    };
+  };
+}
```

Comment:
P1: 타입 지정이 문자열로 되어있습니다...!

```suggestion
interface FetchData {
interface FetchData {
  hourlyPay: number;
  startsAt: string;
  workHour: number;
  description: string;
  closed: boolean;
  shop: {
    item: {
      name: string;
      category: string;
      address1: string;
      description: string;
      imageUrl: string;
      originalHourlyPay: number;
    };
  };
}
```

Reply:
바로 반영하겠습니다